rover_controller:
  ros__parameters:

    robot_base_frame: base_footprint

    twist_topic: cmd_vel

    position_controller_name: "rover_joint_position_controller"
    velocity_controller_name: "rover_joint_velocity_controller"

    wheel_radius: .075
    track_width: 0.38
    wheelbase: 0.40
    
    drive_velocity_limit: 10.0
    steering_angle_limit: 0.785

    publish_rate: 50.0

    steering_joints:
      - left_front_servo_joint
      - right_front_servo_joint
      - left_rear_servo_joint
      - right_rear_servo_joint

    drive_joints:
      - left_front_wheel_joint
      - right_front_wheel_joint
      - left_middle_wheel_joint
      - right_middle_wheel_joint
      - left_rear_wheel_joint
      - right_rear_wheel_joint